homeassistant/components/simplisafe/__init__.py

```diff
@@ -193,25 +196,28 @@ def __init__(self, hass, config_entry, systems):
         """Initialize."""
         self._config_entry = config_entry
         self._hass = hass
+        self.last_event_data = {}
         self.systems = systems
 
+    async def _update_system(self, system):
+        """Update a system."""
+        try:
+            await system.update()
+            latest_event = await system.get_latest_event()
+        except Exception as err:  # pylint: disable=broad-except
```

Comment:
We should at least differentiate between exceptions and catch expected exceptions first and log these separately. Then as fallback end with broad exception and log.